WordPress/src/main/java/org/wordpress/android/WordPress.java

```diff
@@ -446,6 +438,15 @@ public static Blog setCurrentBlog(int id) {
         return currentBlog;
     }
 
+    public static Blog setCurrentBlogAndSetVisible(int id) {
+        Blog blog = getBlog(id);
```

Comment:
In the current version of the code `setCurrentBlogAndSetVisible` and `setCurrentBlog` despite doing almost the same thing they have different code. 
I also noticed that `setCurrentBlog` is very similar to `getBlog`.

My .2$: It's probably better to have `setCurrentBlogAndSetVisible` calling `setCurrentBlog`, and in order, change `setCurrentBlog` that should call `getBlog`.